### PR TITLE
Replace all the horrible mkmsg_*() methods with GoMessageHelper

### PR DESCRIPTION
**Pull request by jerith**
I didn't split up the commits this time because too much stuff uses base classes from other places.

### CODE REVIEW

File: go/vumitools/tests/utils.py [hunk 1]
```diff
@@ -197,12 +197,14 @@ def poll_metrics(self, assert_prefix=None, app=None):
...
         if batch_id is None and conv is not None:
             batch_id = conv.batch.key
         return self.user_api.api.mdb.add_outbound_message(
             msg, batch_id=batch_id)
 
     def store_inbound_msg(self, msg, conv=None, batch_id=None):
+        assert False
         if batch_id is None and conv is not None:
             batch_id = conv.batch.key
         return self.user_api.api.mdb.add_inbound_message(
```
Comment (hodgestar) [inline go/vumitools/tests/utils.py]: Maybe these two methods can just be deleted now?

**Comment by hodgestar** [comment]
Other than deleting the two methods that start with "assert False", I think this is good.

File: go/vumitools/tests/utils.py (hunk 1 above)
Comment (jerith) [inline go/vumitools/tests/utils.py]: Oops. I was supposed to do that.

**Comment by hodgestar** [comment]
:+1: once Travis is happy.

**Pull request merged**